test/test.getch.ts

```diff
@@ -449,11 +449,11 @@ describe('🎏 data handling', () => {
     assert.deepStrictEqual(res.data, {});
   });
 
-  it('replaces application/x-www-form-urlencoded with application/json', async () => {
+  it('application/x-www-form-urlencoded with object data should stringify with qs', async () => {
```

Comment:
I hate asking this, but can you make it read "it should stringify with qs when content-type is set to application/x-www-form-urlencoded"?  Reading the test title broke my brain the first time.